add recovery guide for two dc deployment (#9804)
pingcap/docs: command-line-flags-for-pd-configuration.md

- The address of Prometheus Pushgateway, which does not push data to Prometheus by default.
- Default: `""`

## `--force-new-cluster`

> - Force to create a new cluster using current nodes.

Reviewer: ```suggestion
- Forcibly creates a new cluster using current nodes.
```

- Default: `false`

> - It is recommended to use this flag only when recovering services due to PD losing most replicas, which might cause data loss.

Reviewer: ```suggestion
- It is recommended to use this flag only for recovering services when PD loses most of its replicas, which might cause data loss.
```
